packages/expansion-hub/src/ExpansionHub.ts

```diff
@@ -47,7 +48,46 @@ export interface ExpansionHub extends RevHub {
 
     // Device Control
     getBulkInputData(): Promise<BulkInputData>;
-    getADC(): Promise<number>;
+
+    /**
+     * Get the ADC reading of a given channel.
+     * @param channel
+     * @param rawMode true for raw counts, false for mV
+     */
+    getADC(channel: number, rawMode: boolean): Promise<number>;
+
+    /**
+     * For SystemType.DigitalIO, the result is the total current running through
+     * the ground of the digital GPIO sub-system.
```

Comment:
```suggestion
     * the shared ground of the digital channels.
```

Is my proposal still accurate?

Reply:
Updated the documentation